### PR TITLE
Cpu temp assembly

### PR DESCRIPTION
**Pull request by dmpop**
### Description

Describe the overall goals of this pull request.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

File: articles/CPUTemp.asm.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:trans="http://docbook.org/ns/transclusion"
+  xmlns:its="http://www.w3.org/2005/11/its"
+  xmlns="http://docbook.org/ns/docbook">
+  <resources>
+    <resource xml:id="_cpu-temp-check" href="../tasks/cpu-temp-check.xml"/>
+    <resource xml:id="_bci-gfdl" href="../common/license_gfdl1.2.xml"/>
+  </resources>
+
+  <structure xml:id="minvm-intro" xml:lang="en" renderas="article">
```
Comment (tomschr) [inline articles/CPUTemp.asm.xml:20]: The ID looks... familiar. :wink: Copy-pasta issue?

```suggestion
  <structure xml:id="cpu-temp-check" xml:lang="en" renderas="article">
```

File: DC-task-check-cpu-temperature
Comment (tomschr) [inline DC-task-check-cpu-temperature:1]: Technically, this DC file is correct. However, I'm not sure if this topic should stand on its own or if it should be integrated into a larger, overarching theme  (something like systems administration?)

Probably this is out of scope for this issue; perhaps something for our next SLE writers call? (Could be that I miss some information)

File: articles/CPUTemp.asm.xml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        </revision>
+      </revhistory>
+      <!-- add author's e-mail -->
+      <meta name="maintainer" content="dpopov@suse.com" its:translate="no"/>
+      <!-- ISO date of last update as YYYY-MM-DD -->
+      <meta name="updated" content="2023-05-04" its:translate="no"/>
+      <!-- enter the platform identifier or list of comma-separated list of identifiers -->
+      <meta name="architecture" content="x86_64;zseries;power;aarch64" its:translate="no"/>
+      <meta name="productname">
+        <productname version="ALP">&sles;</productname>
```
Comment (tomschr) [inline articles/CPUTemp.asm.xml:40]: Is ALP a version? :thinking: I would move "ALP" _inside_ the tag like this:

```suggestion
        <productname version="1">ALP</productname>
```

The version (here "1") is taken directly from Tomás' assembly file.

**Review by tomschr — CHANGES_REQUESTED** [review]
@dmpop Thanks Dima! I'm fine with it. Found some minor issues.

File: articles/CPUTemp.asm.xml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            <para>
+              Root permissions to install the required package
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              The package <package>sensors</package>
+            </para>
+          </listitem>
+        </itemizedlist>
```
Comment (dariavladykina) [inline articles/CPUTemp.asm.xml:110]: This part is already featured in the article (after Introduction). This is a duplicate I suggest removing from abstract.

File: articles/CPUTemp.asm.xml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+            <term>WHAT?</term>
+            <listitem>
+              <para>Step-by-step explanation of how to configure CPU temperature monitoring.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+          <term>WHY?</term>
+          <listitem>
+            <para>
+              You want to reduce your electricity bill and make sure that hardware runs optimally.
```
Comment (dariavladykina) [inline articles/CPUTemp.asm.xml:81]:
```suggestion
              You want to reduce your electricity bill and make sure that the hardware runs optimally. 
```

File: tasks/cpu-temp-check.xml [hunk 5]
```diff
@@ -60,30 +60,22 @@
...
+   Checking and monitoring CPU temperature has several benefits.
   </para>
   <itemizedlist>
    <listitem>
-    <para> Energy savings and cost reduction. If all your CPUs are running at full
-     speed, it takes much more energy than if they are idling. Also cooling down
-     CPUs is a critical cost factor, especially in data centers.
+    <para> Energy savings and cost reduction. When a CPU run at full
+     speed, it consumes more energy than when it is idling. Also, running
+     CPUs cool is a critical cost factor, especially in data centers.
```
Comment (dariavladykina) [inline tasks/cpu-temp-check.xml:69]:
```suggestion
    <para>Energy savings and cost reduction. When a CPU runs at full
     speed, it consumes more energy than when it is idling. Also, running
     CPUs cool is a critical cost factor, especially in data centers.
```

File: tasks/cpu-temp-check.xml [hunk 6]
```diff
@@ -60,30 +60,22 @@
...
+     Identifying processes that consume too much CPU power. Doing that can  help to free your CPU resources and increase the CPU's responsiveness.
     </para>
    </listitem>
    <listitem>
-    <para> Better protection of the CPUs and the hardware. CPUs has a thermal protection
-     when the CPU gets too warm. This protects it from overheating. However, if
-     a CPU temperature reaches 80°C or higher, it is a warning sign that
-     something is wrong. It can be that a fan is dead, the cooling system does not
-     work properly, or the thermal paste wasn't applied correctly.
+    <para>Easier detection of cooling issues. If a CPU temperature reaches 80°C or higher, it indicates that there is a problem with the cooling system, a fan, or thermal paste wasn't applied correctly.
```
Comment (dariavladykina) [inline tasks/cpu-temp-check.xml:78]:
```suggestion
    <para>Easier detection of cooling issues. If the CPU temperature reaches 80°C or higher, it indicates that there is a problem with the cooling system or the fan, or that the thermal paste was not applied correctly.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

File: tasks/cpu-temp-check.xml [hunk 7]
```diff
@@ -20,13 +20,13 @@
...
    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
     <dm:bugtracker>
      <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
      <dm:component>Smart Docs</dm:component>
      <dm:product>Documentation</dm:product>
-     <dm:assignee>dmitri.popov@suse.com</dm:assignee>
+     <dm:assignee>dpopov@suse.com</dm:assignee>
     </dm:bugtracker>
     <dm:translation>no</dm:translation>
    </dm:docmanager>
```
Comment (dariavladykina) [inline tasks/cpu-temp-check.xml:32]: This information is maintained at the assembly level only, it needs to be removed.

File: tasks/cpu-temp-check.xml [hunk 8]
```diff
@@ -20,13 +20,13 @@
  xmlns:xlink="http://www.w3.org/1999/xlink">
 
  <info>
-   <title>Checking CPU temperature</title>
+   <title>Checking and monitoring CPU temperature</title>
```
Comment (dariavladykina) [inline tasks/cpu-temp-check.xml:23]: Suggest removing this title as it is superfluous and orphaned for such a short article that doesn't have any other parts.
To replace this: `<module resourceref="_cpu-temp-check" renderas="section"/>` by `<module resourceref="_cpu-temp-check" renderas="section" omittitle="true"/>` to get rid of the duplicated title.

**Review by dariavladykina — COMMENTED** [review]
Please consider some more suggestions as to the article structure.
Also, suggest renaming the assembly - changing the assembly file name.

**Pull request merged**